CHANGELOG.md

```diff
@@ -21,6 +21,11 @@ Deprecated modules
 Deprecated names
 ----------------
 
+* In `Algebra.Morphism.Structures`:
+  ```agda
+  homo  ↦  ∙-homo
+  ```
```

Comment:
This unfortunately is in the wrong section of the CHANGELOG, as it's not a deprecated name. Instead it's a breaking change to the name.

Reply:
Now done... but there *is* still a `WARNING_ON_USE` deprecation warning in `Algebra.Morphism.Structures.IsMagmaHomomorphism` associated with the copy of the old name. Does that not also merit an entry in the `Deprecated names` section?